tripal_chado/includes/TripalImporter/NewickImporter.inc

```diff
@@ -108,7 +108,6 @@ class NewickImporter extends TripalImporter {
       '#maxlength' => 255,
     ];
 
-    $type_cv = tripal_get_default_cv('phylotree', 'type_id');
```

Comment:
is that why the newick importer broke?  I couldn't figure it out, i swore i had it running at the hackathon.